reporter/otlp_reporter.go

```diff
@@ -255,24 +184,24 @@ func (r *OTLPReporter) ReportCountForTrace(_ libpf.TraceHash, _ uint16, _ *Trace
 // ExecutableKnown returns true if the metadata of the Executable specified by fileID is
 // cached in the reporter.
 func (r *OTLPReporter) ExecutableKnown(fileID libpf.FileID) bool {
-	_, known := r.executables.Get(fileID)
+	_, known := r.pdata.Executables.Get(fileID)
```

Comment:
Fix from https://github.com/open-telemetry/opentelemetry-ebpf-profiler/pull/256:
```suggestion
	_, known := r.pdata.Executables.GetAndRefresh(fileID, ...)
```

Reply:
These are rather misleading, because your PR isn't merged yet ...